python/eos/parameter_TEST.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import eos
+
+class StaticMethodTests(unittest.TestCase):
+
+    def test_assert_valid_name_0(self):
+        "Value of 'name' is valid"
```

Comment:
Different notation to `observable_TEST.py`. What's better?

Reply:
You first have to tell me what these pseudo comments mean.